common/src/main/java/com/wynntils/features/overlays/MobTotemTrackingFeature.java

```diff
@@ -63,65 +38,27 @@ protected MobTotemTimerOverlay() {
                             HorizontalAlignment.Right,
                             OverlayPosition.AnchorSection.TopRight),
                     new GuiScaledOverlaySize(120, 35));
-
-            updateTextRenderSetting();
         }
 
         @Override
-        public void render(
-                PoseStack poseStack, MultiBufferSource.BufferSource bufferSource, float partialTicks, Window window) {
-            if (renderTaskCache == null) {
-                updateRenderTaskCache();
-            }
-
-            BufferedFontRenderer.getInstance()
-                    .renderTextsWithAlignment(
-                            poseStack,
-                            bufferSource,
-                            this.getRenderX(),
-                            this.getRenderY(),
-                            renderTaskCache,
-                            this.getWidth(),
-                            this.getHeight(),
-                            this.getRenderHorizontalAlignment(),
-                            this.getRenderVerticalAlignment());
+        public String getTemplate() {
+            return IntStream.rangeClosed(1, Models.MobTotem.getMobTotems().size())
```

Comment:
A bit sad that we can't express this fully as a wynntils function expression. Oh well, one step at a time.

Reply:
Yes, I thought the same.